webpack.dev.js

```diff
@@ -8,8 +8,6 @@ module.exports = merge(common, {
   devtool: 'source-map',
   mode: 'development',
   plugins: [
-    new InjectManifest({
-      swSrc: './src/sw.js'
-    })
+    new CopyWebpackPlugin(['./src/sw.js'])
```

Comment:
Can't decide - would it be clearer to have this as `sw.dev.js` and copy to `sw,js`?

Reply:
IMO this means that it makes no difference:)